elf2uf2/elf2uf2.cpp

```diff
@@ -118,13 +174,26 @@ int realize_page(std::shared_ptr<std::iostream> in, const std::vector<page_fragm
     return 0;
 }
 
+/**
+ * @brief Checks if a specific address is mapped.
+ * 
+ * @param pages Map of pages with their fragments.
+ * @param addr Address to check.
+ * @return true if the address is mapped, false otherwise.
+ */
 static bool is_address_mapped(const std::map<uint32_t, std::vector<page_fragment>>& pages, uint32_t addr) {
     uint32_t page = addr & ~(UF2_PAGE_SIZE - 1);
     if (!pages.count(page)) return false;
-    // todo check actual address within page
+    // TODO: Check actual address within page
     return true;
 }
 
+/**
+ * @brief Generates an absolute UF2 block.
+ * 
+ * @param abs_block_loc Target address for the absolute block.
+ * @return Generated UF2 block.
+ */
```

Comment:
It would probably be good to mention Errata E10 in this comment and the one below - it's not just any absolute block, it's an absolute block required to work around Errata E10 in the datasheet